packages/client/components/ReflectionCard/EmojiUsersReaction.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {EmojiUsersReaction_reactji$key} from '~/__generated__/EmojiUsersReaction_reactji.graphql'
+
+const EmojiUsersReactionParent = styled('div')({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  maxHeight: 300,
+  maxWidth: 280,
+  overflow: 'auto',
+  wordWrap: 'break-word',
+  overflowWrap: 'break-word',
+  whiteSpace: 'normal'
+})
+
+const LIST_FORMATTER =
+  Intl.ListFormat !== undefined
+    ? new Intl.ListFormat('en', {style: 'long', type: 'conjunction'})
+    : {format: (arr: string[]) => arr.join(', ')} // fallback for safari pre Big Sur
+
+interface Props {
+  reactjiRef: EmojiUsersReaction_reactji$key
+}
+
+const EmojiUsersReaction = ({reactjiRef}: Props) => {
+  const reactji = useFragment(
+    graphql`
+      fragment EmojiUsersReaction_reactji on Reactji {
+        id
+        users {
+          id
+          preferredName
+          picture
+        }
+      }
+    `,
+    reactjiRef
+  )
+
+  return (
+    <EmojiUsersReactionParent>
+      {LIST_FORMATTER.format(reactji.users.map(({preferredName}) => preferredName))}
```

Comment:
+1 I'd love to see the reactji type, e.g. `:heart:`, here as well as the user's name. When I see the name of the emoji, it helps me search for the emoji next time with my keyboard rather than needing to scroll to find it 

<img width="209" alt="reactji" src="https://user-images.githubusercontent.com/39854876/193245284-beccae32-2410-4ad8-a04f-18579b090239.png">

Reply:
@nickoferrall good idea! see below an improved copy
<img width="453" alt="Screenshot 2022-10-03 at 16 04 48" src="https://user-images.githubusercontent.com/1017620/193597882-3bd96f47-86d9-40aa-b301-784ec7ea3a05.png">